docs/tables/steampipecloud_organization_member.md

```diff
@@ -16,7 +16,7 @@ from
   steampipecloud_organization_member;
 ```
 
-### List pending members
+### List invited members
```

Comment:
Should we add an invited member example, or did you mean to replace? If so, why is the pending members example not required?